components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/docs/examples/apis/search_get.yaml

```diff
@@ -0,0 +1,86 @@
+- id: searchApis
+  request:
```

Comment:
Shall we add a title: 
Suggestion:
title: Search APIs and Documentation By a Given Query